content/en/blog/2024/inpod-traffic-redirection-ambient/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: "Design Istio Ambient to Work with Any Platforms and CNIs"
+description: An innovative traffic redirection mechanism between workload pods and ztunnel.
+publishdate: 2024-01-26
+attribution: "Ben Leggett (Solo.io), Yuval Kohavi (Solo.io), Lin Sun (Solo.io)"
+keywords: [Ambient,Istio,CNI,ztunnel,traffic]
+---
+
+The Istio project [announced its innovative new ambient mode](/blog/2022/introducing-ambient-mesh/) in 2022, and [released an Alpha implementation](/news/releases/1.18.x/announcing-1.18/#ambient-mesh) in early 2023.
```

Comment:
Already used "innovative" in the sub-title to refer to something else. I suggest: `The Istio project [announced ambient mesh - it's new sidecarless dataplane mode]`

Reply:
updated